ground/gcs/src/plugins/uavobjectbrowser/treeitem.cpp

```diff
@@ -47,7 +47,8 @@ HighLightManager::HighLightManager(long checkingInterval, QTime *currentTime)
 
     if (currentTime == NULL)
         m_currentTime = new QTime;
-    m_currentTime = currentTime;
+
+    *m_currentTime = *currentTime;
```

Comment:
Yeah, I mean why is our member a pointer at all? QTime stores only a single pointer to the private implementation so it's POD/we gain nothing by using pointers. 😃